CheckerGame.py

```diff
@@ -305,7 +311,20 @@ def gameEnd(self, numBoardStates) -> int:
                 return _P2PIECE
         
         return 0
-
+    
+    #create the lines
+    def debug_line(self, lines):
+        self.debug_lines = lines
+
+    def debug(self, window):
```

Comment:
```suggestion
    def debug(self, window, value, debug_message):
```
Could you make it so this function only uses a value and message from the parameter instead of getting in from self.debug_lines